app/models/manageiq/providers/openstack/network_manager/network_router.rb

```diff
@@ -2,10 +2,26 @@ class ManageIQ::Providers::Openstack::NetworkManager::NetworkRouter < ::NetworkR
   include ProviderObjectMixin
   include AsyncDeleteMixin
 
-  supports :create_network_router
-  supports :delete_network_router
-  supports :update_network_router
```

Comment:
`supports :create_network_router` is still used in `ManageIQ::Providers::Openstack::NetworkManager` can this be removed from the NetworkManager?

`:delete_network_router` and `:update_network_router`  is not used anymore. So those can be removed from `SupportsFeatureMixin`